### PR TITLE
IOSXE/show wireless stats client delete reasons

### PR DESCRIPTION
**Pull request by qduk**
## Description
Parser created for iosxe show wireless stats client delete reasons

## Screenshots:
Capture

## Checklist:
- [x] I have updated the changelog.
- [x] All new and existing tests passed.
- [x] All new code passed compilation.

### CODE REVIEW

**Review by ThomasJRyan — APPROVED** [review]

File: src/genie/libs/parser/iosxe/tests/ShowWirelessStatsClientDeleteReasons/cli/equal/golden_output_expected.py [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+            "4way_handshake_failure___m1_issue": 0,
+            "4way_handshake_failure___m3_issue": 0,
+            "exclusion_policy_template_fail": 0,
+            "dot11_cipher_suite_rejected": 0
+        },
+        "informational_delete_reason": {
+            "mobility_wlan_down": 0,
+            "ap_upgrade": 0,
+            "l3_authentication_failure": 0,
+            "ap_down/disjoin": 2,
```
Comment (tahigash) [inline src/genie/libs/parser/iosxe/tests/ShowWirelessStatsClientDeleteReasons/cli/equal/golden_output_expected.py:148]: please replace `/` to `_`

File: src/genie/libs/parser/iosxe/tests/ShowWirelessStatsClientDeleteReasons/cli/equal/golden_output_expected.py [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+            "client_not_allowed_by_assisted_roaming": 0,
+            "wired_client_deleted_due_to_wgb_delete": 0,
+            "client_abort": 0,
+            "mobility_peer_delete": 0,
+            "no_ip": 469,
+            "bssid_down": 1,
+            "dot11_qos_policy": 0,
+            "roam_across_policy_profile_deny": 0,
+            "4way_handshake_failure___m1_issue": 0,
+            "4way_handshake_failure___m3_issue": 0,
```
Comment (tahigash) [inline src/genie/libs/parser/iosxe/tests/ShowWirelessStatsClientDeleteReasons/cli/equal/golden_output_expected.py:140]: `4way_handshake_failure___m1_issue` to `4way_handshake_failure_m1_issue` (1x underscore)

File: src/genie/libs/parser/iosxe/tests/ShowWirelessStatsClientDeleteReasons/cli/equal/golden_output_expected.py [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+            "ip_update_timeout": 0,
+            "sae_commit_received_in_associated_state": 0,
+            "nack_ifid_mismatch": 0,
+            "eogre_invalid_vlan": 0,
+            "eogre_empty_domain": 0,
+            "eogre_invalid_domain": 0,
+            "eogre_domain_shut": 0,
+            "eogre_invalid_gateway": 0,
+            "eogre_all_gateways_down": 0,
+            "eogre_flex___no_active_gateway": 0,
```
Comment (tahigash) [inline src/genie/libs/parser/iosxe/tests/ShowWirelessStatsClientDeleteReasons/cli/equal/golden_output_expected.py:117]: same 1x `_`

**Review by tahigash — CHANGES_REQUESTED** [review]
thank you for your contribution. reviewed schema and left some comments.

File: src/genie/libs/parser/iosxe/tests/ShowWirelessStatsClientDeleteReasons/cli/equal/golden_output_expected.py (hunk 1 above)
Comment (qduk) [reply src/genie/libs/parser/iosxe/tests/ShowWirelessStatsClientDeleteReasons/cli/equal/golden_output_expected.py]: Updated

File: src/genie/libs/parser/iosxe/tests/ShowWirelessStatsClientDeleteReasons/cli/equal/golden_output_expected.py (hunk 2 above)
Comment (qduk) [reply src/genie/libs/parser/iosxe/tests/ShowWirelessStatsClientDeleteReasons/cli/equal/golden_output_expected.py]: Updated

File: src/genie/libs/parser/iosxe/tests/ShowWirelessStatsClientDeleteReasons/cli/equal/golden_output_expected.py (hunk 3 above)
Comment (qduk) [reply src/genie/libs/parser/iosxe/tests/ShowWirelessStatsClientDeleteReasons/cli/equal/golden_output_expected.py]: Updated

**Review by tahigash — APPROVED** [review]
schema looks good now. thanks for addressing comments!

**Pull request merged**